### DISCUSSION TITLE
Windows 11 webcam source is showing but no image

### DISCUSSION THREAD

**Discussion by dingerr** [ptrumpis/snap-camera-server#116] [Q&A]
so basically about a week or two ago I had to reset my computer. after downloading / installing the latest snap server, I noticed that the camera (both my integrated one, and external one) were showing up as options, but the image is not showing...
I have tried uninstalling / reinstalling / updating all my webcams drivers, as well as uninstalling and reinstalling snapcam, docker, everything. My camera image is showing up though for other programs such as OBS, and zoom. But for snapcam, (latest version), again the sources show up but no image comes from them. I've made sure that no other apps are the webcams as well. 
I would appreciate any help or tips so much, I feel like im going crazy over here... lol.
i really appreciate any and all suggestions / responses...

heres a quick vid of what it looks like on my side: https://github.com/ptrumpis/snap-camera-server/assets/95573944/bfebf686-0230-4aca-b7c3-17daa6b58fc7

much love @ptrumpis

**Comment by ptrumpis** [comment]
Check your Windows "**Camera Privacy Settings**"

![CameraPrivacySettings](https://github.com/ptrumpis/snap-camera-server/assets/116500225/f595d36a-8a9b-47a5-b018-4f45d2694d33)

**Comment by dingerr** [comment]
I have made sure it is turned on and should be alllowed to use the webcam /cams. Ill attach another video. 
thank you @ptrumpis 

https://github.com/ptrumpis/snap-camera-server/assets/95573944/b3eb5fcd-8e94-4a3a-9b87-157e27edeffe
